### ISSUE TITLE
Update the documentation for the ens, NEP, and NMEP configuration options

### ISSUE THREAD

**Issue by JohnHalleyGotway** [dtcenter/MET#2513]
## Describe the Task ##
In dtcenter/METplus#2125, @rogerhar notes that NEP and NMEP appear in the documentation for both Gen-Ens-Prod [section 9.2.1](https://met.readthedocs.io/en/latest/Users_Guide/gen-ens-prod.html?highlight=nmep#ensemble-forecasts-derived-from-a-set-of-deterministic-ensemble-members) and Ensemble-Stat [section 5.3.1](https://met.readthedocs.io/en/latest/Users_Guide/config_options.html?highlight=nmep#ensemblestatconfig-default). The entire `ens` dictionary was removed from Ensemble-Stat in MET version 11.0.0 but more documentation updates are needed to clarify this. Sections [5.3.1.1](https://met.readthedocs.io/en/latest/Users_Guide/config_options.html?highlight=nmep#ens), [5.3.1.2](https://met.readthedocs.io/en/latest/Users_Guide/config_options.html?highlight=nmep#nbrhd-prob), and [5.3.1.3](https://met.readthedocs.io/en/latest/Users_Guide/config_options.html?highlight=nmep#nmep-smooth) should be deleted, but only after confirming that the content has been properly migrated to the Gen-Ens-Prod chapter.

### Time Estimate ###
4 hours?

### Sub-Issues ###
Consider breaking the task down into sub-issues.
None needed.

### Relevant Deadlines ###
*List relevant project deadlines here or state NONE.*

### Funding Source ###
*Define the source of funding and account keys here or state NONE.*

### Assignee ###
- [ ] Select **engineer(s)** or **no engineer** required
- [ ] Select **scientist(s)** or **no scientist** required

### Labels ###
- [x] Select **component(s)**
- [x] Select **priority**
- [x] Select **requestor(s)**

### Projects and Milestone ###
- [x] Select **Repository** and/or **Organization** level **Project(s)** or add **alert: NEED PROJECT ASSIGNMENT** label
- [x] Select **Milestone** as the next official version or **Future Versions**

## Define Related Issue(s) ##
Consider the impact to the other METplus components.
- [x] [METplus](https://github.com/dtcenter/METplus/issues/new/choose), [MET](https://github.com/dtcenter/MET/issues/new/choose), [METdataio](https://github.com/dtcenter/METdataio/issues/new/choose), [METviewer](https://github.com/dtcenter/METviewer/issues/new/choose), [METexpress](https://github.com/dtcenter/METexpress/issues/new/choose), [METcalcpy](https://github.com/dtcenter/METcalcpy/issues/new/choose), [METplotpy](https://github.com/dtcenter/METplotpy/issues/new/choose)
No impacts.

## Task Checklist ##
See the [METplus Workflow](https://metplus.readthedocs.io/en/latest/Contributors_Guide/github_workflow.html) for details.
- [ ] Complete the issue definition above, including the **Time Estimate** and **Funding Source**.
- [ ] Fork this repository or create a branch of **develop**.
Branch name: `feature_<Issue Number>_<Description>`
- [ ] Complete the development and test your changes.
- [ ] Add/update log messages for easier debugging.
- [ ] Add/update unit tests.
- [ ] Add/update documentation.
- [ ] Push local changes to GitHub.
- [ ] Submit a pull request to merge into **develop**.
Pull request: `feature <Issue Number> <Description>`
- [ ] Define the pull request metadata, as permissions allow.
Select: **Reviewer(s)** and **Development** issues
Select: **Repository** level development cycle **Project** for the next official release
Select: **Milestone** as the next official version
- [ ] Iterate until the reviewer(s) accept and merge your changes.
- [ ] Delete your fork or branch.
- [ ] Close this issue.

**Comment by j-opatz** [comment]
After review, it looks like an update is only needed to correct the configuration file name in Sect 5.3.1.1 through 5.3.1.3. It currently lists those subsections as part of Ensemble-Stat, when they should be Gen-Ens-Prod. So a simple update to 5.3.1 to be Gen-Ens-Prod, followed by the creation of a new 5.3.2 section for Ensemble-Stat, should work. The following subsections (5.3.1.4+) are correctly listed as Ensemble-Stat and should be placed under the new 5.3.2 section. 

No information will need to be deleted during this migration.

**Comment by j-opatz** [comment]
Documentation was updated as described, with an additional move of the ensemble_flag dictionary to be included in the new GenEnsProd configuration section.
